packages/backend/server/routes/stats.get.ts

```diff
@@ -0,0 +1,49 @@
+import { useBinding } from "~/utils/bucket";
+
+export default eventHandler(async (event) => {
+  try {
+    const binding = useBinding(event);
+
+    let truncated = true;
+    let cursor: string | undefined;
+    let totalObjects = 0;
+    const orgs = new Set<string>();
+    const packages = new Set<string>();
+
+    while (truncated) {
+      const list = await binding.list({ cursor });
+
+      totalObjects += list.objects.length;
+
+      for (const object of list.objects) {
+        try {
+          const [org, pkg] = object.key.split(":");

```

Comment:
this is not correct! you should slice the usePackagesBucket.base first! and also use that prefix so you only see packages. 

read the code of #287 carefully!